src/shared/ui/SubPageHeader/index.test.tsx

```
@@ -0,0 +1,27 @@
+    };
+    const { getByText } = renderSubPageHeader(props);
+
+    expect(getByText(props.title)).not.toBeNull();
+  });
+
+  it('renders button', () => {
+    const props: SubPageHeaderProps = {
+      backLink: undefined,
+      title: '',
+    };
+    const {} = renderSubPageHeader(props);
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)